### ISSUE TITLE
The Impact of Batch Size

### ISSUE THREAD

**Issue by AmingWu** [ucbdrive/few-shot-object-detection#50]
I find the batch size has a very large impact on your model. Could you give an interpretation?

**Comment by thomasehuang** [comment]
It depends how you are training with the different batch sizes. [This](https://github.com/ucbdrive/few-shot-object-detection/issues/7) issue should be relevant. If you follow the schedule when modifying the batch size, the only difference should be the training time, and the results should be similar.

**Comment by AmingWu** [comment]
Thank you very much. I have three GPUs. Each GPU is put one image. How should I adjust the learning rate and training steps for this case？

**Comment by AmingWu** [comment]
Thank you very much. I have solved this problem.
